pkgs/dart_mcp_server/bin/main.dart

```diff
@@ -65,6 +71,19 @@ void main(List<String> args) async {
 
 final argParser =
     ArgParser(allowTrailingOptions: false)
+      ..addOption(
+        dartSdkOption,
+        help:
+            'The path to the root of the desired Dart SDK. Defaults to the '
+            'DART_SDK environment variable.',
+      )
+      ..addOption(
+        flutterSdkOption,
```

Comment:
if the Flutter SDK is provided, do we want to always use the Dart SDK included with the Flutter SDK? I could see danger of using a Dart SDK provided by the CLI arg that is incompatible with the Flutter SDK provided by the Flutter CLI arg.